aten/src/ATen/StorageImpl.h

```diff
@@ -112,4 +110,7 @@ struct AT_API StorageImpl : public Retainable {
   }
 };
 
+namespace detail {
+AT_API Backend get_backend(StorageImpl* storage_impl);
+}
```

Comment:
NB: at some point we're going to have to move this out of this file. (Doesn't have to be this PR).